test/new-e2e/tests/windows/install-test/assert.go

```diff
@@ -19,6 +19,21 @@ import (
 	"testing"
 )
 
+// AssertEqualPathf compares two paths case insensitively, ignoring trailing slashes
+func AssertEqualPathf(t *testing.T, expected string, actual string, fmt string, args ...any) bool {
+	t.Helper()
+
+	// Normalize paths
+	// trim trailing slashes
+	expected = strings.TrimSuffix(expected, `\`)
+	actual = strings.TrimSuffix(actual, `\`)
+	// windows paths are case-insensitive
+	expected = strings.ToLower(expected)
+	actual = strings.ToLower(actual)
+
+	return assert.Equalf(t, expected, actual, fmt, args...)
+}
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities. -->
**💭 thought**
I would change this to have a `TrimTrailingSlashesAndLower(path string)` and then have an explicit assert in the callee
```go
assert.Equal(t, TrimTrailingSlashesAndLower(expected), TrimTrailingSlashesAndLower(axctual), ...)
```